### DISCUSSION TITLE
Support translating Kirby `tags` fields

### DISCUSSION THREAD

**Discussion by tideg** [kirby-tools/community#19] [Q&A] [answered]
Is it possible to also auto-translate a tags field? I saw that it´s not in the list for the supported fields but maybe there is an option to enable it anyways?

**Comment by johannschopplich** [comment, marked as answer]
Kirby tags fields are now supported. Please update to [v2.4.0](https://github.com/kirby-tools/kirby-content-translator/releases/tag/v2.4.0). Thanks!

**Comment by tideg** [comment]
Great, thank you very much @johannschopplich!

**Comment by metakolor** [comment]
The current way of handling kirbytags which could not be parsed is via 
`<div translate="no">(mytag: )</div>.
`
Wouldn't it make more sense to use 
`<span> `
for this?

**Reply by johannschopplich** [reply]
Good question. Technically, it doesn't matter. The plugin configures DeepL to [ignore a specific tag](https://developers.deepl.com/docs/xml-and-html-handling/xml#ignored-tags). This tag doesn't have to be semantic. A non-semantic tag like `<x>` would be equally effective. But yeah, a span might be a touch more readable. 

Edit: I just changed it.
